python/chronos/src/bigdl/chronos/model/tcn.py

```diff
@@ -43,8 +43,7 @@
 import torch
 import torch.nn as nn
```

Comment:
To make the models consistant and easier to add new models (tf or pytorhc), rename the file to tcn_pytorch as the other two models, or just move all of them to model/pytorch and remove the  file suffix "pytorch".